backstop/scripts/ready.js

```diff
@@ -116,12 +116,15 @@ module.exports = async function (page, scenario, vp) {
 
   // All the alerts on the page must be processed.
   try {
-    await page.waitForFunction("jQuery('.mass-alerts-block:not([data-alert-processed])').length === 0", {
-      timeout: 60 * 1000,
+    await page.waitForFunction("document.querySelectorAll('.mass-alerts-block').length == document.querySelectorAll('.mass-alerts-block[data-alert-processed]').length", {
+      timeout: 6000,
```

Comment:
@justafish Just to double-check, since the previous timeout is one minute, is this new timeout now supposed to be six seconds?

Reply:
@tom-fleming it is not - thanks for catching that :laughing:

Reply:
@tom-fleming yeah 6 seconds seems more than enough here from my tests - but can readjust if needed?